### PR TITLE
Customers controller To Mailer

### PR DESCRIPTION
**Pull request by StanislasBASQUIN**
- Adding the Customer Controller
- Fixing Signout NavBar
- Feature Follow - UnFollow
- Connection Mailer Customer
- Adding first name and last name to customer model

### CODE REVIEW

File: app/views/business_follows/create.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>BusinessFollows#create</h1>
+<p>Find me in app/views/business_follows/create.html.erb</p>
```
Comment (mvoland) [inline app/views/business_follows/create.html.erb:2]: controller#create should not have associated view

Comment (Vivien-Ploix) [reply app/views/business_follows/create.html.erb:2]: Ah yes, it must have been created when we generated the controller, thanks !

**Comment by StanislasBASQUIN** [comment]
[Créer les Mails pour l'inscription des nouveaux Customer et Owner](https://trello.com/c/J65juzoN/118-cr%C3%A9er-les-mails-pour-linscription-des-nouveaux-customer-et-owner)

**Comment by StanislasBASQUIN** [comment]
[BusinessFollow : Création du bouton Follow Business](https://trello.com/c/5TYPGf3q/94-businessfollow-cr%C3%A9ation-du-bouton-follow-business)

**Comment by StanislasBASQUIN** [comment]
[BusinessFollow : Création du bouton Follow Business](https://trello.com/c/5TYPGf3q/94-businessfollow-cr%C3%A9ation-du-bouton-follow-business)

**Pull request merged**